app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,166 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskServiceTest {
+
+    private static final String TASK_TITLE = "test";
+    private static final String UPDATE_POSTFIX = "!!!";
+
+    private TaskService taskService;
+
+    @BeforeEach
+    void setUp() {
+        // subject
+        taskService = new TaskService();
+
+        //fixtures
+        Task task = new Task();
+        task.setTitle(TASK_TITLE);
+
+        taskService.createTask(task);
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks{
+        @Nested
+        @DisplayName("Task가 등록되어 있다면")
+        class Context_have_task{
+            @Test
+            @DisplayName("등록된 모든 Task를 리턴한다")
+            void It_return_tasks(){
+                List<Task> tasks = taskService.getTasks();
+
+                assertThat(tasks).hasSize(1);
+
+                Task task = tasks.get(0);
+                assertThat(task.getTitle()).isEqualTo(TASK_TITLE);
+            }
+        }
```

Comment:
저기 위에 있는 `@BeforeEach`를 여기 `Context` 안쪽으로 옮겨오셔야 컨텍스트가 말이 됩니다.

```suggestion
    @BeforeEach
    void setUp() {
        // subject
        taskService = new TaskService();
    }

    @Nested
    @DisplayName("getTasks 메소드는")
    class Describe_getTasks{
        @Nested
        @DisplayName("Task가 등록되어 있다면")
        class Context_have_task{
        
            // 설명문의 "Task가 등록되어 있다면" 과 똑같은 조건이 되도록 여기에서 준비해 줍니다.
            @BeforeEach
            void prepareTask() {
                Task task = new Task();
                task.setTitle(TASK_TITLE);

                taskService.createTask(task);
            }

            @Test
            @DisplayName("등록된 모든 Task를 리턴한다")
            void It_return_tasks(){
                List<Task> tasks = taskService.getTasks();

                assertThat(tasks).hasSize(1);

                Task task = tasks.get(0);
                assertThat(task.getTitle()).isEqualTo(TASK_TITLE);
            }
        }
```